CorsixTH/Lua/hospitals/player_hospital.lua

```diff
@@ -481,6 +481,12 @@ function PlayerHospital:onEndDay()
     end
   end
 
+  for _, staff in ipairs(self.staff) do
+    if staff:isIdle() then
+      self.world.dispatcher:answerCall(staff)
+    end
+  end
+
```

Comment:
This looks like it is preventing handymen from responding to calls -- I'm guessing they're never considered idle.

I have checked with latest master and this bug isn't present there.